src/content/apis/comics/superhero.md

```diff
@@ -0,0 +1,12 @@
+---
+name: SuperHero API
+description: Get all SuperHeroes and Villians data from all universes under a single API.
+categories:
+  - comics
+  - entertainment
+features:
+  https: true
+  pricing: upgradable
```

Comment:
```suggestion
  pricing: free
```
Upon looking into the link you shared, it seems the API is entirely free, isn't it?

Reply:
@TheLearneer looking at the pricing here : https://rapidapi.com/jakash1997/api/superhero-search/pricing, it seems there will be request limit and thus can be upgraded. Please look into it once. Otherwise will update as `free`.

Reply:
@TheLearneer looking at the pricing here : https://rapidapi.com/jakash1997/api/superhero-search/pricing, it seems there will be request limit and thus can be upgraded. Please look into it once. Otherwise will update as `free`.

Reply:
There are no mention of API plans in the official document itself.
So it might be possible that the RapidAPI hosted version has the rate-limit and not the actual endpoint itself?

Reply:
Seems so. Will update.